config.yaml

```diff
@@ -17,6 +17,13 @@ options:
       Backend site name, e.g. mybackend.local. If none given, will
       work out from the configured 'backend' config above.
     default: ""
+  cache_all:
+    type: boolean
+    description: >-
```

Comment:
should this use the `|` character rather than `>-`?

Reply:
Iirc, the thing is that `>` will be fold the newline characters into spaces. Since this is later published in churmhub, `>`would probably be better to avoid formatting issues.